competitions/template/ansible/inv/local/group_vars/all.tmpl

```diff
@@ -0,0 +1,71 @@
+---
+competition_name: <COMPETITIONNAME>
```

Comment:
In this template, is it the case that all the things the deployer needs to edit are "`<IN ANGLE BRACKETS>`" ?

If so, would be good to point that out at the top.

Reply:
I actually point this out in the README.md.  The reason I don't want to put this in the actual template file is that the template file has comments that are meant to pass through to the target competition after someone has made it from the template.